src/main/java/de/tum/cit/aet/artemis/quiz/dto/participation/StudentQuizParticipation.java

```diff
@@ -0,0 +1,5 @@
+package de.tum.cit.aet.artemis.quiz.dto.participation;
+
+public interface StudentQuizParticipation {
```

Comment:
Rename this to `StudentQuizParticipationDTO` for now. We'll think of a possible better naming later.